sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -19,7 +19,7 @@
 
 ### Features Added
 - Added support for [Vector Search](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch.md).
-- Added support for [Semantic Search](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples/Sample08_SemanticSearch.md).
+- Added support for [Semantic Search](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/samples).
```

Comment:
Did you intend to add the link to all the samples here? Somehow looking at the semantic search I was expecting to get linked into a semantic search specific sample.

Reply:
The main branch is currently targeting the preview version. In the preview, there was no sample for semantic search, which is why I have temporarily added the samples link here. We have included this sample for the GA release. In my next PR, I will add back all the changes and samples that are identical to those in the GA. I will also include this link back here specifically for semantic search.